docs/controllers.md

```diff
@@ -9,3 +11,14 @@ The OpenStackClusterStackRelease controller’s main responsibilities are:
 - Update the OpenStackClusterStackRelease status to `ready` once all related OpenStackNodeImageReleases are also `ready`
 
 ![OSCSR controller](./images/openstackclusterstackrelease-controller.png "OSCSR controller")
```

Comment:
I suggest modifying this img according to https://github.com/SovereignCloudStack/cluster-stack-provider-openstack/pull/92

Reply:
fixed: c380e5fa26ad5d4725148dfaf08bed0f842afcc8